### PR TITLE
Adding Read-Only Dashboard to Sky Island

### PR DESCRIPTION
**Pull request by njenan**
Adding a read-only dashboard to sky island, available at "http://<your server:your port>/".  Shows basic status of the system and a list of all current jails.

Note that I am dealing with an issue on my end where I'm not able to get Sky-Island running, so I haven't fully integration tested this gui.  I will update this PR as soon as I'm able to test it out, but I opened this just for informational purposes and review.

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  <!-- /.container -->
+
+
+  <!-- Bootstrap core JavaScript
+    ================================================== -->
+  <!-- Placed at the end of the document so the pages load faster -->
+  <script src="./script.js"></script>
+</body>
+
+</html>
```
Comment (briandowns) [inline static/index.html]: Can you add a new line here?

File: static/script.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            };
+        });
+
+        updateJailsTemplates();
+    });
+
+    setTimeout(pollJails, 60 * 1000);
+}
+
+pollJails();
```
Comment (briandowns) [inline static/script.js]: Can you add a new line here?

File: static/style.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+h3.bold {
+    font-weight: bold;
+}
+
+.asoftime {
+    font-family: monospace;
+    color: grey;
+}
```
Comment (briandowns) [inline static/style.css]: Can you add a new line here?

File: static/index.html [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+  <meta name="description" content="">
+  <meta name="author" content="">
+
+  <title>Starter Template for Bootstrap</title>
```
Comment (briandowns) [inline static/index.html]: Is it possible to update this to something "sky island" related?

File: static/index.html [hunk 5]
```diff
@@ -0,0 +1,211 @@
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+  <meta name="description" content="">
+  <meta name="author" content="">
```
Comment (briandowns) [inline static/index.html]: Put your name here! :D

**Comment by njenan** [comment]
Corrected the issues.

**Comment by briandowns** [comment]
Sweet! Thanks for this!

**Pull request merged**